endpoint-sec-ui-tests/tests/ui/test_client_cannot_outlive_handler.stderr

```diff
@@ -0,0 +1,5 @@
+error[E0432]: unresolved import `endpoint_sec`
+ --> tests/ui/test_client_cannot_outlive_handler.rs:1:5
+  |
+1 | use endpoint_sec::Client;
```

Comment:
That, erm, feels like the wrong error to check :D

Reply:
Yeah I was telling myself "wait, how does it work in a new crate"